src/chemistry/mozart/mo_gas_phase_chemdr.F90

```diff
@@ -779,20 +779,21 @@ subroutine gas_phase_chemdr(lchnk, ncol, imozart, q, &
     !-----------------------------------------------------------------------
 
     esfact = 1._r8
-    call shr_orb_decl( calday, eccen, mvelpp, lambm0, obliqr  , &
-         delta, esfact )
-
-    !-----------------------------------------------------------------
-    !	... lookup the photolysis rates from table
-    !-----------------------------------------------------------------
-    call table_photo( reaction_rates, pmid, pdel, tfld, zmid, zint, &
-                      col_dens, zen_angle, asdir, cwat, cldfr, &
-                      esfact, vmr, invariants, ncol, lchnk, pbuf )
-
-    !-----------------------------------------------------------------
-    !	... get calculated photolysis rates from TUV-x
-    !-----------------------------------------------------------------
-    call tuvx_get_photo_rates( ncol )
+    call shr_orb_decl( calday, eccen, mvelpp, lambm0, obliqr, delta, esfact )
+
+    !if (tuvx_active) then
+       !-----------------------------------------------------------------
+       !	... get calculated photolysis rates from TUV-x
+       !-----------------------------------------------------------------
+       call tuvx_get_photo_rates( ncol )
+    !else
+       !-----------------------------------------------------------------
+       !	... lookup the photolysis rates from table
+       !-----------------------------------------------------------------
+       call table_photo( reaction_rates, pmid, pdel, tfld, zmid, zint, &
+            col_dens, zen_angle, asdir, cwat, cldfr, &
+            esfact, vmr, invariants, ncol, lchnk, pbuf )
+    !endif
```

Comment:
What happens when we implement a 3rd scheme for photolysis?  Rather than using the `tuvx_active` logical here, should we use a higher-level selector?